IFC4x3/Properties/s/Strike_2QDp8uDwH5yOXoaLbKFWum/Documentation.md

```diff
@@ -0,0 +1 @@
+The geographic direction of a line created by the intersection of a plane and the horizontal.
```

Comment:
The angle is between true north and the created line? Measured CCW or CW?

Reply:
Geographic directions are (usually?) measured CW from North.